.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# they will be requested for review when someone opens a pull request.
+*	@haroldsphinx @segunjkf @calvin-puram @voltagebots
```

Comment:
Lets create a Github Group and add every under..its easier to manage that way

Reply:
@haroldsphinx makes sense, but i don't know if i have enough permission to do that.

Reply:
There is a @blockopsnetwork/devops  group

Reply:
Done. Thanks @voltagebots